src/components/molecules/plan/plan.scss

```
@@ -0,0 +1,62 @@
+@use "../../variables" as *;
+.plan {
+  padding: 0;
+
+  .header-box {
+    display: flex;
+    justify-content: center;
+    padding: 0;
+    background-color: $orange_zorba;
+    font-family: $fontfamily-gothic;
+    font-weight: 700;
+    font-size: $fontsize-l;
+    line-height: 29px;
+    color: white;
+    padding: 20px 0px;
+  }
+
+  .box-list1 {
+    background-color: $gray_desertStorm;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 24px 0px;
+  }
+  .box-list1-text {
+    font-family: $fontfamily-gothic;
+    font-size: $fontsize-xxl;
+    font-weight: 700;
+    line-height: 58px;
+    color: $orange_bistre;
+    margin-bottom: 16px;
+  }
+  .box-list1-text::first-letter {
+    font-size: $fontsize-l;
+    line-height: 29px;
+    margin-right: 4px;
+  }
+  .box-list1-text-bottom {
+    font-size: $fontsize-s;
+    line-height: 20px;
+  }
+
+  .box-list2 {
+    display: flex;
+    justify-content: center;
+    padding: 20px 10px;
+    font-size: $fontsize-s;
+    border-top: 1px solid $gray_gainsboro;
+    border-right: 1px solid $gray_gainsboro;
+    border-left: 1px solid $gray_gainsboro;
+  }
+}
+.box-list2:nth-child(6n) {
+  border-bottom: 1px solid $gray_gainsboro;
+}
```

```suggestion
  .box-list2 {
    display: flex;
    justify-content: center;
    padding: 20px 10px;
    font-size: $fontsize-s;
    border-top: 1px solid $gray_gainsboro;
    border-right: 1px solid $gray_gainsboro;
    border-left: 1px solid $gray_gainsboro;
    
    &:nth-child(6n) {
      border-bottom: 1px solid $gray_gainsboro;
    }
  }
}
```
scssの記法を使うとこんな感じにもかけます！

また、一番下だけボーダーを付けたいという観点では `:last-child` を使ったほうが直感的かもしれません。